packages/contracts/test/token/erc20/governance-erc20.ts

```diff
@@ -374,5 +383,401 @@ describe('GovernanceERC20', function () {
+          context('`to` receives via transfer from `from`', async () => {
+            beforeEach(async () => {
+              await token.mint(from.address, 100);
+              await expect(
+                token.connect(from).transfer(to.address, 100)
+              ).to.not.emit(token, 'DelegateChanged');
+            });
+
+            it('`from` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(0);
+            });
+            it('`from`s delegate has not changed', async () => {
+              expect(await token.delegates(from.address)).to.equal(
+                from.address
+              );
+            });
+            it('`from`s delegate has the correct voting power', async () => {
+              const fromDelegate = await token.delegates(from.address);
+              expect(await token.getVotes(fromDelegate)).to.equal(0);
+            });
+
+            it('`to` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(0);
+            });
+            it('`to`s delegate has not changed', async () => {
+              expect(await token.delegates(to.address)).to.equal(toDelegate);
+            });
+            it('`to`s delegate has the correct voting power', async () => {
+              expect(await token.getVotes(toDelegate)).to.equal(100);
+            });
+          });
+        });
+
+        context('`to` has not delegated before', async () => {
+          context('`to` receives via `mint` from `address(0)`', async () => {
+            beforeEach(async () => {
+              await expect(token.mint(to.address, 100))
+                .to.emit(token, 'DelegateChanged')
+                .withArgs(to.address, toDelegate, to.address); // the mint triggers automatic self-delegation
+              toDelegate = to.address;
+            });
+
+            it('`to` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(100);
+            });
+            it('`to`s delegate has not changed', async () => {
+              expect(await token.delegates(to.address)).to.equal(toDelegate);
+            });
+            it('`to`s delegate has the correct voting power', async () => {
+              expect(await token.getVotes(toDelegate)).to.equal(100);
+            });
+          });
+
+          context('`to` receives via transfer from `from`', async () => {
+            beforeEach(async () => {
+              await token.mint(from.address, 100);
+              await expect(token.connect(from).transfer(to.address, 100))
+                .to.emit(token, 'DelegateChanged')
+                .withArgs(to.address, addressZero, to.address); // the transfer triggers automatic self-delegation
+              toDelegate = to.address;
+            });
+
+            it('`from` has the correct voting power', async () => {
+              expect(await token.getVotes(to.address)).to.equal(100);
```

Comment:
test doesn't match the description

Reply:
Fixed in c0d3b11a35f333f3c0dac00e73c4f7f2edb89620